.github/workflows/publish.yml

```diff
@@ -18,6 +18,12 @@ jobs:
           bundler-cache: true
           ruby-version: ruby
       - uses: rubygems/release-gem@v1
+      - name: Create a GitHub release
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          bundle exec rake create_release_notes
+          gh release create $(git tag --points-at @) --notes-file relnotes.md
```

Comment:
ask) Should we need to create a release version tag before this?

Reply:
Yes. It’s created and pushed by line 20. And also, it should run _before_ the `git push` below which would cause `git tag --points-at @` to no longer return the current tag 😅

Reply:
Thank ypu for the explanation! I understand😀